src/renderer/components/stake/add/AddStake.tsx

```diff
@@ -147,36 +146,55 @@ export const AddStake: React.FC<Props> = ({
     })
   }, [onStake, asset, runeAsset, assetAmountToStake, runeAmountToStake])
 
-  const prevFeeRef = useRef<O.Option<BaseAmount>>(O.none)
-
   const formatFee = useCallback(
-    (fee: BaseAmount) =>
+    (fee: BaseAmount, asset) =>
       formatAssetAmountCurrency({
         amount: baseToAsset(fee),
-        asset: O.toUndefined(oChainAsset),
+        asset,
         trimZeros: true
       }),
-    [oChainAsset]
+    []
```

Comment:
as it does not depend on anything we could move it to the helper file

Reply:
ok